CONTRIBUTING.md

```diff
@@ -113,6 +113,54 @@ yarn lint:fix
 yarn build
 ```
 
+## Setting up your Developer Bot
+
+Head to the [Discord Developer Portal](https://discord.com/developers/applications), and create a new application (call it something like "AO Dev Test"), and optionally a testing server in Discord.
+
+In the portal, click "Bot" and add a bot. You'll need to enable Server Members Intent, and Message Content Intent like so:
+
+<div align="center">
+  <figure>
+    <img src="./assets/bot-intents.png" alt="Bot Intents Settings"/>
+  </figure>
+</div>
+
+You'll also want to copy your token, and save it someplace safe (like GitHub Secrets).
+
+Next, in `OAuth2->General`, you'll want to grab your Client ID and Client Secret for safekeeping as well.
+
+Add a redirect, and point it to `http://localhost:3000/api/auth/callback/discord` (if you're hosting on a VPS with Remote Environment, use your VPS IP in place of localhost)
+
+Change the authorization method to "In-app Authorization", checking the `bot` and `applications.commands` scopes.
+
+You'll want to enable the following bot permissions:
+
+- `Manage Server`
+- `Create Instant Invite`
+- `Read Messages/View Channels`
+- `Send Messages`
+- `Create Public Threads`
+- `Send Messages in Threads`
+- `Manage Threads`
+- `Embed Links`
+- `Read Message History`
+- `Add Reactions`
+- `Use Slash Commands`
+
+When you're done, it should look like this:
+
+<div align="center">
+  <figure>
+    <img src="./assets/OAuth2-Settings.png" alt="OAuth2 Settings"/>
+  </figure>
+</div>
+
+You can then generate a link from `OAuth2->URL Generator` using the `bot` and `applications.commands`, with the same permissions.
+
+To make this quicker for you, you can use: `https://discord.com/oauth2/authorize?client_id=YOUR_ID&permissions=328565083201&scope=bot+applications.commands`, which has the correct permissions integer set. Just fill in your client ID, and you'll be ready to go!
```

Comment:
Maybe lets link to https://discordapi.com/permissions.html here?

Reply:
Add to existing content, or replace 158-160?

Reply:
Lets add to existing content, say "replace your id in this link or use this generator"